content/applications/hr/recruitment.rst

```diff
@@ -1,10 +1,809 @@
+:guilabel:`Additional emails` line.
+
+If any attachments need to be added, click the :guilabel:`Attachments` button and a file explorer
+window appears. Navigate to the desired file, then click :guilabel:`Open` to attach it to the email.
+The attachment loads, and is listed above the :guilabel:`Attachments` button.
+
+If the emailed interview must be completed by a specific date, select a date for the
+:guilabel:`Answer deadline` in the lower-right area of the pop-up window. Click on the line next to
+:guilabel:`Answer deadline`, and a calendar selector appears. Use the :guilabel:`< (left)` and
+:guilabel:`> (right)` arrows on either side of the month to navigate to the desired month, then
+click on the :guilabel:`day` to select the date.
+
+The :guilabel:`Mail template` field is pre-populated based on the configuration for the interview. A
+different template can be chosen from the drop-down menu if desired. If a new template is selected,
+the new email template loads in the email body.
+
+To send the email with the interview link to the applicant, click :guilabel:`Send` at the bottom of
+the email pop-up.
+
+.. image:: recruitment/send-survey.png
+   :align: center
+   :alt: Send a custom survey, also referred to as an interview form, to an applicant using a
+         pre-configured templates.
+
+Refuse
+~~~~~~
+
+At any point in the recruitment pipeline, an applicant can be refused from te job application
+process. To refuse an applicant, click on an applicant's card to navigate to a detailed view of the
+applicant's card. At the top of the applicant's card, there are several buttons. Click the one
+labeled :guilabel:`Refuse`.
+
+A :guilabel:`Refuse Reason` pop-up appears. Select the :guilabel:`Refuse Reason` from the drop-down
+menu. The default refuse reasons in Odoo are: :guilabel:`Doesn't fit the job requirements`,
+:guilabel:`The applicant is not interested anymore`, and :guilabel:`The applicant gets a better
+offer`. These three refusal reasons have pre-configured email templates associated with them.
+Additional refusal reasons can be created and existing ones can be modified or deleted. To add a new
+refusal reason, type in the name of the reason, then click :guilabel:`Create and Edit...`. A
+:guilabel:`Create: Refuse Reason` pop-up window appears. Type in the name of the new email template
+in the :guilabel:`Email Template` field, and click :guilabel:`Create and Edit...`. A
+:guilabel:`Create: Email Template` window loads. Configure the new email template, then click
+:guilabel:`Save`.
+
+Email templates in Odoo can use dynamic placeholders, which are customized pieces of data that
+populate dynamic content. For example, the applicant's name can be a piece of dynamic content. The
+applicant's name will appear anytime that dynamic placeholder appears on the email template. For
+more detailed information on email templates, refer to :doc:`this documentation
+<../general/email_communication/email_template>`.
```

Comment:
Funny you mention the manual placeholder- I didn't think I was implying they could change anything, just explaining what a dynamic placeholder was, and how they are used, with an example to really get the point across. I'll explain that they are not manual changes in 15. I believe I originally only added this info because it was suggested  to explain wat exactly a dynamic placeholder was.

Reply:
I mentioned these are for "pre-configured" email templates and they MIGHT/MAY contain dynamic placeholders- then I explain what that is. i think that's more clear.